pace-util/pace/util/zarr_monitor.py

```diff
@@ -243,6 +267,12 @@ def _get_attrs(self, quantity):
             **quantity.attrs,
         }
 
+    def _order_dims(self, quantity):
+        if self.array is None:
+            return quantity
+        else:
+            dim_order = self.array.attrs["_ARRAY_DIMENSIONS"]
```

Comment:
Oh cool, didn't know you can get them like this. Is this public API in zarr?

Reply:
Ha whoops this function was a leftover from a previous approach, will delete. But to answer your question, `zarr.core.Array.attrs` is public API (as a MutableMapping), so I think this is fine, and I do use this to check and match dim orders above.